src/wp-admin/css/dashboard.css

```diff
@@ -659,6 +503,44 @@ body #dashboard-widgets .postbox form .submit {
 	margin: 0;
 }
 
+.classicpress-news-inline-notice {
+	background-color: #f6f7f7;
+    padding: 12px;
+    margin-bottom: 10px;
+    border-bottom: 1px solid #f0f0f1;
```

Comment:
Can these be indented with tabs rather than spaces?

Reply:
I will get this fixed. I don't use spaces, but it could be copy/pasting from devtools.